### PR TITLE
Improve the vagrant flow

### PR DESCRIPTION
**Pull request by staritza**
This pr allows a user to run redash locally or test without having to run any manual steps, except the run command itself. We also updated the docs to include this change.

To run locally, run ./bin/vagrant_ctl.sh start 
To test, ./bin/vagrant_ctl.sh test

### CODE REVIEW

**Comment by arikfr** [comment]
Looks like you need to rebase with updated master.

File: docs/dev/vagrant.rst [hunk 1]
```diff
@@ -14,27 +14,11 @@ To get started with this box:
     `Vagrant <https://www.vagrantup.com/>`__ installed.
 2.  Clone the Re:dash repository:
     ``git clone https://github.com/getredash/redash.git``.
-3.  Change dir into the repository (``cd redash``) and run run
-    ``vagrant up``. This might take some time the first time you run it,
+3.  Change dir into the repository (``cd redash``)
+4a. To execute tests, run ``./run_tests.sh``
+4b. To run the app, run ``./run_redash_locally.sh``.
```
Comment (arikfr) [inline docs/dev/vagrant.rst]: Notice the docs mention the old scripts and not `vagrant_ctl`.

File: setup/vagrant/provision.sh [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+cd /opt/redash/current
+cp /opt/redash/.env /opt/redash/current
+cd /opt/redash/current/rd_ui
+bower install
+cd /opt/redash/current
+
+#install requirements
+sudo pip install -r /opt/redash/current/requirements_dev.txt
+sudo pip install -r /opt/redash/current/requirements.txt
+sudo pip install pymongo==3.2.1
```
Comment (arikfr) [inline setup/vagrant/provision.sh]: It's better to run `sudo pip install -r /opt/redash/current/requirements_all_ds.txt` here.

File: docs/dev/vagrant.rst (hunk 1 above)
Comment (staritza) [inline docs/dev/vagrant.rst]: My bad. Good catch!

File: setup/vagrant/provision.sh (hunk 2 above)
Comment (staritza) [inline setup/vagrant/provision.sh]: Running requirements_all_ds.txt has some dependencies that are not able to be installed. I could spend some time trying to fix all the dependencies. But they are not really required for anything on the vagrant box as far as I can tell. So I just decided to go with installing pymongo, which is the only thing we really need (one test fails when we dont have it). I could be wrong though? The dependencies can be sorted out in a later PR? Is not using requirements_all_ds.txt introducing a regression?

**Comment by staritza** [comment]
Please hold off on this pr for a while. We are trying to work on our team git/redash work flow.

**Comment by staritza** [comment]
@arikfr you can ignore this pr. we created [this](https://github.com/getredash/redash/pull/1044) with only the commits that matter and changes to reflect your comments

**Pull request closed without merging**